### ISSUE TITLE
Insure PDF accessibility

### ISSUE THREAD

**Issue by eldobbins** [acep-uaf/aetr-web-book-2024#68]
Yuri says the PDF cannot be posted on the website if it isn't accessible.  This is likely a UAF requirement.

This tool will check it: https://helpx.adobe.com/acrobat/using/create-verify-pdf-accessibility.html.  Oh it isn't a web-based tool. It is the instructions for how to use Adobe Pro to do it.

**Comment by eldobbins** [comment]
UA no longer seems to offer Adobe Pro.  We have FoxIt instead, which can also check accessibility. https://www.uada.edu/employees/division-accessibility/table-chart-accessibility/foxit-table.aspx ...

## Summary
The checker found problems which may prevent the document from being fully accessible.
- Needs manual check:2
- Passed manually:0
- Failed manually:0
- Skipped:1
- Passed:11
- Failed:18

But I don't even understand what they are talking about in the "failed" messages. And Quarto documentation's only reference to accessibility is `fig-alt`, so maybe it isn't a priority for them?  I think this is going to be a heavy lift.

Need clarification from Yuri.

**Comment by eldobbins** [comment]
https://github.com/quarto-dev/quarto-cli discussions 9333. Money quote:

> I feel like this is something that Pandoc has struggled with ...  I'm crossing my fingers that the Typyt Team will get there soon (https://github.com/typst/typst issues 133), though!

**Comment by eldobbins** [comment]
Sent email to Yuri to ask about this.

**Comment by eldobbins** [comment]
Response from Yuri was to provide her with a copy in Word and she would do the work to insure accessibility.  But all the issues documented in #108 apply, so at this time, I am giving up on releasing this report as an official ACEP report.
